### PR TITLE
Add KeebsPCB pre-Alpha support

### PR DESCRIPTION
**Pull request by Gondolindrim**
## Description

This PR adds support for the [pre-revision Alpha version of the KeebsPCB](https://github.com/Gondolindrim/KeebsPCB/releases/tag/pre-Alpha). A revised Alpha version is underway, which firmware will be added later.

The PCB offers a hotswap feature. A single layout is available, consisting of a default ANSI layout with 7U bottom row.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/acheron/keebspcb/config.h [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x4150 // AP for AcheronProject
+#define PRODUCT_ID      0x4b45 // KE for Keebs
```
Comment (noroadsleft) [inline keyboards/acheron/keebspcb/config.h]: This should be uppercase on the value.

```suggestion
#define PRODUCT_ID      0x4B45 // KE for Keebs
```

Comment (Gondolindrim) [reply keyboards/acheron/keebspcb/config.h]: Comitted in c336178

File: keyboards/acheron/keebspcb/info.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+    "keyboard_name": "KeebsPCB", 
+    "url": "http://gondolindrim.github.io/AcheronDocs/keebs/intro.html", 
+    "maintainer": "Gondolindrim", 
+    "width": 15, 
+    "height": 5, 
+    "layouts": {
+        "LAYOUT": {
```
Comment (noroadsleft) [inline keyboards/acheron/keebspcb/info.json]:
```suggestion
        "LAYOUT_all": {
```

Comment (noroadsleft) [inline keyboards/acheron/keebspcb/info.json]: I just realized this keyboard is `LAYOUT_60_ansi_tsangan`.

```suggestion
        "LAYOUT_60_ansi_tsangan": {
```

File: keyboards/acheron/keebspcb/keebspcb.h [hunk 3]
```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include "quantum.h"
+
+#define ___ KC_NO
+
+#define LAYOUT_all( \
```
Comment (noroadsleft) [inline keyboards/acheron/keebspcb/keebspcb.h]:
```suggestion
#define LAYOUT_60_ansi_tsangan( \
```

File: keyboards/acheron/keebspcb/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+   [0] = LAYOUT_all(
```
Comment (noroadsleft) [inline keyboards/acheron/keebspcb/keymaps/default/keymap.c]:
```suggestion
   [0] = LAYOUT_60_ansi_tsangan(
```

File: keyboards/acheron/keebspcb/keymaps/default/keymap.c [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+        KC_GESC, KC_1   , KC_2   , KC_3   , KC_4   , KC_5   , KC_6   , KC_7   , KC_8   , KC_9   , KC_0   , KC_MINS, KC_EQL , KC_BSPC,
+        KC_TAB , KC_Q   , KC_W   , KC_E   , KC_R   , KC_T   , KC_Y   , KC_U   , KC_I   , KC_O   , KC_P   , KC_LBRC, KC_RBRC, KC_BSLS, 
+        KC_CAPS, KC_A   , KC_S   , KC_D   , KC_F   , KC_G   , KC_H   , KC_J   , KC_K   , KC_L   , KC_SCLN, KC_QUOT, KC_ENT ,
+        KC_LSFT, KC_Z   , KC_X   , KC_C   , KC_V   , KC_B   , KC_N   , KC_M   , KC_COMM, KC_DOT , KC_SLSH, KC_RSFT,        
+        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC ,                   KC_RALT, MO(1)  , KC_RCTL                            ),
+   [1] = LAYOUT_all(
```
Comment (noroadsleft) [inline keyboards/acheron/keebspcb/keymaps/default/keymap.c]:
```suggestion
   [1] = LAYOUT_60_ansi_tsangan(
```

File: keyboards/acheron/keebspcb/rules.mk [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output on port C6
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
+HD44780_ENABLE = no         # Enable support for HD44780 based LCDs
+
+# Enter lower-power sleep mode when on the ChibiOS idle thread
+OPT_DEFS += -DCORTEX_ENABLE_WFI_IDLE=TRUE
```
Comment (noroadsleft) [inline keyboards/acheron/keebspcb/rules.mk]:
```suggestion
OPT_DEFS += -DCORTEX_ENABLE_WFI_IDLE=TRUE

LAYOUTS = 60_ansi_tsangan
```

**Comment by Gondolindrim** [comment]
@noroadsleft suggestions comitted in da041fc

**Comment by zvecr** [comment]
Thanks!

**Pull request merged**